README.adoc

```diff
@@ -0,0 +1,56 @@
+= htsjdk-next-beta
+:toc: macro
+
+image:http://img.shields.io/badge/language-java-brightgreen.svg[Language, link=https://www.java.com]
+image:https://img.shields.io/badge/license-BSD%203--Clause-blue.svg[License (3-Clause BSD), link=https://opensource.org/licenses/BSD-3-Clause]
+
+Staging ground for the development of the next major version of HTSJDK
+
+toc::[]
+
+== Status
+
+[options="header", cols=2, caption=""]
```

Comment:
I would probably put these as a single horizontal line or as part of the badge line.  This table looks nice but takes up most of the vertical space.

Reply:
Done